docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -0,0 +1,710 @@
+      "status": {
+        "idle": 183,
+        "claiming": 2,
+        "running": 19
+      }
+    },
+    "alerting:.es-query": {
+      "count": 225,
+      "status": {
+        "idle": 225,
+      }
+    },
+    "actions:.index": {
+      "count": 89,
+      "status": {
+        "idle": 24,
+        "running": 2,
+        "failed": 63
+      }
+    },
+    "alerting:xpack.uptime.alerts.monitorStatus": {
+      "count": 87,
+      "status": {
+        "idle": 74,
+        "running": 13
+      }
+    },
+  },
+  "schedule": [ # <2>
+    ["10s", 38],
+    ["1m", 101],
+    ["90s", 55],
+    ["5m", 89],
+    ["20m", 62],
+    ["60m", 106],
+    ["1d", 61]
+  ],
+  "overdue": 0, # <5>
+  "estimated_schedule_density": [  # <3>
+    10, 1, 0, 10, 0, 20, 0, 1, 0, 1,
+    9, 0, 3, 10, 0, 0, 10, 10, 7, 0,
+    0, 31, 0, 12, 16, 31, 0, 10, 0, 10,
+    3, 22, 0, 10, 0, 2, 10, 10, 1, 0
+  ]
+}
+--------------------------------------------------
+<1> there are 2,191 tasks in the system
+<2> the scheduled tasks are distributed across a variety of cadences
+<3> the schedule density shows that we expect to exceed the default 10 concurrent tasks
+
+We can infer several important attributes of our workload from this output:
+
+* There are many tasks in our system and ensuring these tasks run on their scheduled cadence will require attention to the {kib} Task Management throughput.
+* Assessing the high frequency tasks (tasks that recurr at a cadence of a couple of minutes and below), we have a need to support a throughput of _approximately_ 400 tasks per minute (38 every 10 seconds + 101 every minute + 55 every 90 seconds).
+* Assessing the medium frequency tasks (tasks that recurr at a cadence of an hour or less), we have a need to support an additional throughput of over 2000 tasks per hour (89 every 5 minutes, + 62 every 20 minutes + 106 each hour). We can naively average these throughput the hour and count these tasks as an additional 30 to 40 tasks per minute.
+* Assessing the estimated schedule density, there are cycles that are due to run upwards of 31 tasks concurrently, but along side these cycles, there are empty cycles. We can expect Task Manager to load balance these tasks throughout the empty cycles, but this won't leave much capacity to handle spikes in fresh tasks that might be scheduled in the future.
+
+These _rough_ calculations give us a lower bound to the required throughput which is that of _at least_ 440 tasks per minute to ensure recurring tasks are excuted, more or less, at their scheduled time. This throughput doesn't account for non recurring tasks that might have been scheduled, nor does it account tasks (recurring or otherwise) that might be scheduled in the future.
+
+Given these inferred attributes, it would be safe to assume that a single {kib} instance with default settings **would not** provide the required throughput. That said, it is quite possible that scaling horizontally by adding a couple more {kib} instances will.
```

Comment:
```suggestion
Given these inferred attributes, it would be safe to assume that a single {kib} instance with default settings **would not** provide the required throughput. It is possible that scaling horizontally by adding a couple more {kib} instances will.
```